packages/ilib-loctool-webos-c/test/CFile.test.js

```diff
@@ -883,6 +883,21 @@ describe("cfile", function() {
         var set = cf.getTranslationSet();
         expect(set.size()).toBe(4);
     });
+    test("CFileTest4", function() {
+        expect.assertions(2);
+
+        var cf = new CFile({
+            project: p,
+            pathName: "./t4.c",
+            type: cft
+        });
+        debugger;
+        expect(cf).toBeTruthy();
+        cf.extract();
+
+        var set = cf.getTranslationSet();
+        expect(set.size()).toBe(1);
+    });
```

Comment:
How about adding another test:

```
test("CFileTest5", function() {
        expect.assertions(3);

        var cf = new CFile({
            project: p,
            pathName: "./t4.c",
            type: cft
        });
        debugger;
        expect(cf).toBeTruthy();
        cf.extract();

        var set = cf.getTranslationSet();
        var resources = set.getAll();
        expect(resources.length).toBe(1);
        expect(resources[0].source).toBe("* You can watch TV even while updating the software.")l
    });
```

Reply:
The reason is because I don't know if you extracted the first (empty) string or the second string. Either way, your change will get only 1 string (good!) but it may get the wrong string (bad!).